crates/libs/sys/Cargo.toml

```diff
@@ -374,3 +374,7 @@ Win32_UI_WindowsAndMessaging = ["Win32_UI"]
 Win32_UI_Wpf = ["Win32_UI"]
 Win32_UI_Xaml = ["Win32_UI"]
 Win32_UI_Xaml_Diagnostics = ["Win32_UI_Xaml"]
+
+# These features are unstable and require the nightly Rust compiler:
+raw_dylib = []
```

Comment:
If you're confident in your compatibility layer and want to merge this feature; you could opt for putting it behind a `--cfg windows_nightly` option instead of a feature flag. Such a flag wouldn't be prone to transitive activation through dependencies, and it would allow people to more easily experiment with unstable features and avoid the bitrot associated of maintaining a branch.

See e.g. [`tokio_unstable` for reference](https://docs.rs/tokio/latest/tokio/#unstable-features).

Reply:
Ooh, that looks handy - thanks! Features really aren't great for this kind of thing.